aequilibrae/project/network/network.py

```diff
@@ -426,3 +427,44 @@ def __count_items(self, field: str, table: str, condition: str) -> int:
         with commit_and_close(connect_spatialite(self.project.path_to_file)) as conn:
             c = conn.execute(f"select count({field}) from {table} where {condition};").fetchone()[0]
         return c
+
+    def build_pt_preload(
+            graph, start_time: int, end_time: int, default_pce: float = 1.0, inclusion_cond: str ="start"
+            ) -> dict[(int, int), int]:
```

Comment:
ideally this should be a `np.array` of the correct size to be immediately summable with the load vector created during assignment.

Reply:
Changed to this